sqltest/sql_test_runner.go

```diff
@@ -873,7 +874,7 @@ func (st *sqlTest) doLoadData(require *require.Assertions, command string, noWai
 	}
 	err := kafka.IngestRows(fakeKafka, dataset.sourceInfo, dataset.colTypes, dataset.rows, encoder)
 	require.NoError(err)
-	if !noWait {
+	if !noWait && !dataset.sourceInfo.OriginInfo.IngestOnFirstMV {
```

Comment:
This shouldn't be necessary - it seems like you don't want to wait for data to be loaded in the source? If so, just add the no wait suffix on the command.

Reply:
Done